sdk/storage/azblob/zc_page_blob_client.go

```diff
@@ -70,7 +70,7 @@ func (pb PageBlobClient) WithSnapshot(snapshot string) PageBlobClient {
 // WithVersionID creates a new PageBlobURL object identical to the source but with the specified snapshot timestamp.
 // Pass "" to remove the version returning a URL to the base blob.
 func (pb PageBlobClient) WithVersionID(versionID string) PageBlobClient {
-	p := NewBlobURLParts(pb.URL())
+	p, _ := NewBlobURLParts(pb.URL())
```

Comment:
We should check all of these errors

Reply:
This for sure wouldn't be a problem. that's why I didn't check it. Problem occurs when customer provides us URL directly. In that case we do parsing error check inside the NewBlobURLParts() method.